apis/duck/v1alpha1/addressable_types.go

```diff
@@ -26,15 +26,18 @@ import (
 
 // Addressable provides a generic mechanism for a custom resource
 // definition to indicate a destination for message delivery.
-// (Currently, only hostname is supported, and HTTP is implied. In the
-// future, additional schemes may be supported, and path components
-// ala UI may also be supported.)
-
+// Hostname is deprecated as of v0.6 and replaced with URL. It will be removed in ~v0.7
+// URL string currently supports scheme(http or https), authority(host and optional port), and path
+// Example: http://myhost:8080/ResourceGroup/ResourceName (http://<authority>/<path>)
```

Comment:
At a minimum scheme and host are required.

Reply:
Will add it to the comment

Reply:
Since I replaces it with a struct, I'll remove the comment.